api/filter/http/gzip.proto

```diff
@@ -50,29 +50,24 @@ message Gzip {
 
   // Set of strings that allows specifying which mime-types yield compression; e.g.
   // application/json, text/html, etc. When this field is not specified, compression will be applied
-  // to the following MIME-TYPES: html, text, css, js, json, svg, xml. xhtml".
+  // to the following MIME-TYPES: html, text, css, js, json, svg, xml, xhtml.
```

Comment:
If you say mime types, you should use the correct name, e.g. text/html instead of just html.